lib/index.js

```diff
@@ -14,6 +14,7 @@ let api = function Bitvavo() {
   let apiKey = ''
   let apiSecret = ''
   let accessWindow = 10000
+  let csrf = ''
```

Comment:
Maybe name this `csrfToken`? Because now it's unclear whether it's the cookie header, or a csrf-related option.